services/property-svc/repos/property_value_repo.sql

```diff
@@ -0,0 +1,35 @@
+-- name: CreatePropertyValue :exec
+INSERT INTO property_values
+	(id, property_id, subject_id, text_value, number_value, bool_value, date_value, date_time_value, select_value)
+VALUES (
+        @id,
+        @property_id,
+        @subject_id,
+        sqlc.narg('text_value'),
+        sqlc.narg('number_value'),
+        sqlc.narg('bool_value'),
+        sqlc.narg('date_value'),
+        sqlc.narg('date_time_value'),
+        sqlc.narg('select_value')
+);
+
+-- name: GetPropertyValueBySubjectIDAndPropertyID :one
```

Comment:
Unify this and GetPropertyValueByID and GetPropertyValuesBySubjectID into one query. If you want to you can only unify the latter two, as this query returns only the id, in that case rename this query to GetPropertyValueID.

Reply:
`GetPropertyValuesBySubjectID` now also embeds `property` now, not only the `propertyValue`. I consider this as resolved.